### ISSUE TITLE
Github stars are not updated

### ISSUE THREAD

**Issue by tollerm** [Malabarba/paradox#118]
Github stars in the paradox list are not updated. For example, Magit in the paradox list has 1999 stars while on the github site it already acquired 2086 stars.

**Comment by Malabarba** [comment]
Yeah. I no longer have access to the computer that was fetching these numbers. I'll have to set it up on a proper server.

**Comment by doolio** [comment]
In addition installed packages are not automatically starred when `paradox-automatically-star` is set to `t` or even when executing `paradox-star-all-installed-packages`. Related to this issue?
